finda-auth/src/main/kotlin/finda/findaauth/global/security/passport/filter/PassportFilter.kt

```
@@ -0,0 +1,89 @@
+
+        filterChain.doFilter(request, response)
+    }
+
+    private fun resolvePassport(passportHeader: String?): Passport {
+        if (passportHeader.isNullOrBlank()) {
+            throw InvalidTokenException
+        }
+
+        return try {
+            PassportParser.parseAndValidate(passportHeader, passportProperties.secretKey)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)